compiler/tests-common/tests/org/jetbrains/kotlin/jvm/compiler/AbstractWriteSignatureTest.kt

```diff
@@ -220,7 +230,7 @@ abstract class AbstractWriteSignatureTest : CodegenTestCase() {
             ).joinToString("\n") { "// $it" }
         }
 
-        val expectationRegex = Regex("^// (class|method|field): *([^:]+)(::(.+))? *(//.*)?")
+        val expectationRegex = Regex("^// (class|method|field|!method): *([^:]+)(::(.+))? *(//.*)?")
```

Comment:
Is the new expectation kind used somewhere? I can't find usages in the tests in this PR

Reply:
You are completely right! That's a remnant from earlier testing. The absence of the generated main is now confirmed through the bytecode tests without patterns, detecting a duplicate JVM signature in case it IS generated when it shouldn't.